src/Content/LockException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Exception\LogicException;
+
+/**
+ * @package   Kirby Content
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class LockException extends LogicException
```

Comment:
`Suggestion:` rename to `LockedContentException` to fit more to our other exception classes and be a bit more descriptive.

Reply:
I'm fine with that.